cdc/sink/sink_manager.go

```diff
@@ -30,7 +30,7 @@ import (
 // and backendSink.
 // Manager is thread-safe.
 type Manager struct {
-	bufSink      *bufferSink
+	sink         Sink
```

Comment:
Perhaps we could remove sink_manager and move the relevant fields to processor.

Reply:
I guess if we merge it will increase the complexity? Is the processor already complex enough on its own?